docs/content/guides/developer/nft/asset-tokenization.mdx

```diff
@@ -55,7 +55,7 @@ Select a module to view its details:
 
 <Tabs groupId="modules">
 
-<TabItem label="tokenized_asset" value="tokenized_asset">
+<TabItem label="`tokenized_asset`" value="tokenized_asset">
```

Comment:
This doesn't render correctly.
<img width="209" height="67" alt="image" src="https://github.com/user-attachments/assets/1a55c0f8-b0fe-4245-8966-d77f968e8e88" />